src/components/AlertMessage.tsx

```diff
@@ -0,0 +1,23 @@
+import { Alert, Snackbar } from "@mui/material";
+import { useState } from "react";
+import { AlertMessageProps } from "../Types/AlertMessageProps";
+
+const AlertMessage: React.FC<AlertMessageProps> = ({ message, severityType }) => {
+    const [alertOpen, setAlertOpen] = useState(true);
```

Comment:
we like to use
```suggestion
    const [alertOpen, setAlertOpen] = useState<boolean>(true);
```
change it everywhere in the project